tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts

```diff
@@ -356,8 +356,8 @@ Polymer({
 
             // Update the tooltip position and value
             d3.select(parent.$.tooltip)
-                .style('left', d3.event.pageX + 10 + 'px')
-                .style('top', d3.event.pageY - 10 + 'px')
+                .style('left', (~~d3.select(this).attr('x') + 50) + 'px')
```

Comment:
Also I presume `50` is the width of the tooltip? Can we just measure the bounding client rect instead?

Reply:
The width of the tooltip is 200. I empirically adjusted the padding and find 50 works.

Reply:
I would advise against using such "magic" number. Is it at all impossible to calculate this value based on dimension somewhere?

Reply:
I would imagine d3.event.pageX and d3.event.pageY should just work or d3.select(this).attr('x') + rect.width + 5 and d3.select(this).attr('y') + rect.height + 5 should work. But neither works, except for these magic numbers.